### ISSUE TITLE
How to add more profiles?

### ISSUE THREAD

**Issue by Mrlong2** [joshjcarrier/minecontrol#34]
I could probably rig up some complicated .bat launcher, but I'd prefer to do it the easy way.  How can I do?

**Comment by gizmocodes** [comment]
I agree. Whats the point of a Profile Tab when you can't add a profile? :D

**Comment by GizmoCodes** [comment]
I think I found out how to add a profile. You go to your Home directory (C:/Users/YO_NAME/) and go to .minecontrol2 and copy from [controllers.default] down to the bottom and paste it again, renaming default to something else. gonna try it now

**Comment by GizmoCodes** [comment]
nope, i was wrong.

**Comment by joshjcarrier** [comment]
@gizmodocodes Just to tease, clearly
